learn.qmd

```diff
@@ -1,13 +1,82 @@
 ---
-title: "Learn"
-listing:
-  contents: learn
-  sort: "date desc"
-  type: default
-  categories: true
-  sort-ui: false
-  filter-ui: false
-  feed: true
-page-layout: full
-title-block-banner: true
+format:
+  html:
+    toc: true
 ---
+
+# Learning materials
+
+Our trainings (1) facilitate how to find, choose, and use
+R packages for Outbreak Analytics tasks and (2) extending our R toolset
+to produce open, reproducible and sustainable data analysis projects
+following good Research Sofware practices. These two goals constitute
+the **Outbreak Analytics curriculum** and the **Research Sofware
+curriculum**.
```

Comment:
Here I suggest a one-line edit to a [semantic line breaks](https://sembr.org/) text. Do you prefer to use this consistently across this document?

```suggestion
We organise our training materials in two curricula. The **Outbreak Analytics curriculum** aims to facilitate finding, choosing, and using R packages for outbreak analytics tasks to inform decision-making. The Research **Sofware curriculum** aims to extend our R toolset to produce open, reproducible and sustainable data analysis projects following good Research Sofware practices. 
```